api/src/main/java/io/minio/PresignedPutObjectArgs.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+/** Argument class of MinioClient.presignedPutObject(). */
+public class PresignedPutObjectArgs extends ObjectArgs {
```

Comment:
`presignedPutObject` and `presignedGetObject` seem to be just wrappers over a single method/API `getPresignedObjectUrl`

If this understanding is correct, I don't think we need three different Args classes. A single args class `GetPresignedObjectUrlArgs` should suffice.

Reply:
@anjalshireesh  You point is valid but Argument class is always in sync with method name so i have used different argument class. @balamurugana  need your input on this